### PR TITLE
Add description to `AnimationNodeStateMachine.replace_node`

### PR DESCRIPTION
**Pull request by Anutrix**
Added description to AnimationNodeStateMachine.replace_node.

### CODE REVIEW

File: doc/classes/AnimationNodeStateMachine.xml [hunk 1]
```diff
@@ -143,6 +143,7 @@
 			<param index="0" name="name" type="StringName" />
 			<param index="1" name="node" type="AnimationNode" />
 			<description>
+				Replaces the specified animation node with a new animation node.
```
Comment (AThousandShips) [inline doc/classes/AnimationNodeStateMachine.xml:146]:
```suggestion
				Replaces the given animation node with a new animation node.
```
The methods above use this language so should match

**Review by AThousandShips — APPROVED** [review]
One little detail and otherwise good

File: doc/classes/AnimationNodeStateMachine.xml (hunk 1 above)
Comment (Anutrix) [reply doc/classes/AnimationNodeStateMachine.xml]: Done.

**Comment by Anutrix** [comment]
Is anything blocking this?

**Comment by AThousandShips** [comment]
Not that I know of, but we're in the holiday season right now and mainly merging bug fixes when we do merge 🙂,

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**